front/app/components/FormBuilder/components/FormBuilderSettings/MultiselectSettings/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+// components
+import {
+  Toggle,
+  Box,
+  IconTooltip,
+  Input,
+  Label,
+} from '@citizenlab/cl2-component-library';
+import styled from 'styled-components';
+
+// intl
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+import { Controller, useFormContext } from 'react-hook-form';
+
+const StyledLabel = styled(Label)`
+  height: 100%;
+  margin-top: auto;
+  margin-bottom: auto;
+`;
+
+type Props = {
+  minimumSelectCountName: string;
+  maximumSelectCountName: string;
+  selectCountToggleName: string;
+  selectOptionsName: string;
+};
+
+const MultiselectSettings = ({
+  minimumSelectCountName,
+  maximumSelectCountName,
+  selectCountToggleName,
+  selectOptionsName,
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const { control, setValue, trigger, watch } = useFormContext();
+
+  return (
+    <Box mb="24px">
+      <Box mb="16px">
+        <Controller
```

Comment:
Was it not possible to reuse the `components/HookForm/Toggle` component?

Reply:
Yes you're completely right - I missed that. Thanks :)